protocol-definitions/MC.yaml

```diff
@@ -955,3 +955,12 @@ methods:
           doc: |
             true if the session is found & closed,
             false otherwise.
+  - id: 37
+    name: demoteDataMember
+    since: 2.8
+    doc: |
+      Demotes the data member it's called on to a lite member.
+    request:
+      retryable: false
+      partitionIdentifier: -1
+    response: {}
```

Comment:
how do we communicate success and failure? If communicate failure with exceptions, can you document what are possible exceptions as well?

Reply:
Exception is thrown in case of a failure. Added to the doc part.